futurenhs.app/UI/scss/modules/card/_card.scss

```diff
@@ -61,6 +61,10 @@ $card-hero-padding: 10px;
         margin-bottom: $nhsuk-spacing-point-2;
         max-width: 60%;
         align-self: center;
+
+        &_discussion-title {
+            max-width: 100%;
```

Comment:
Is this a discussion specific bug @simplybenuk? or can we just set the max width for the header text to 100%? I wonder if its a throw back to before we had placeholder images

if we're happy to have all card titles being full width then we could just modify the max width above.

Reply:
@tim-hunt happy to have all cards 100% max width. I just wondered if the 60% was put in because of the group image, so making that change may impact the group cards?